pos_session_pay_invoice/readme/ROADMAP.rst

```diff
@@ -0,0 +1,5 @@
+* Cannot pay invoices in a different currency than that defined in the journal
+  associated to the payment method used to pay/collect payment.
+
+* Should depend on `pos_invoicing` but it requires a refactoring of `pos_invoicing`.
+  It will be improved when migrating to 13.0
```

Comment:
This part is obsolete, as the invoices are now marked as paid.

Reply:
removed